projects/v2/src/app/static/config.ts

```diff
@@ -11,1148 +11,318 @@ export const SHEET_ID = '1tK916JyG5ZSXW_cXfsyZnzXfjyoN-8B2GXLbYD6_vF0';
 
 export const version1Url = 'https://hubmapconsortium.github.io/ccf-releases/v1.0/asct-b/';
 
-export const SHEET_CONFIG: SheetDetails[] = [
+export const PLAYGROUND_SHEET_OPTIONS: SheetOptions[] = [
```

Comment:
There is still a ton of information in here that should be in json/assets. This config.ts needs to go away and instead be encompassed in SheetConfigService.